e2e/commands_test.go

```diff
@@ -276,7 +276,15 @@ func TestBundle(t *testing.T) {
 	assert.Assert(t, fs.Equal(tmpDir.Join("simple.dockerapp"), manifest))
 }
 
-func TestDockerAppLifecycle(t *testing.T) {
+func TestDockerAppLifecycleWithBindMounts(t *testing.T) {
```

Comment:
You can use [subtests](https://blog.golang.org/subtests) (or [this link](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks) here:
```go
func TestDockerAppLifecycle(t *testing.T){
     t.Run("WithBindMounts", func(t *testing.T) {
         testDockerAppLifecycle(t, true)
     }
     t.Run("WithoutBindMounts", func(t *testing.T) {
         testDockerAppLifecycle(t, false)
     }
}
```

Reply:
Get an error when trying to do this: `invalid name "TestDockerAppLifecycle/WithoutBindMounts". Names must be [a-zA-Z0-9-_]+`

Reply:
Alright, `withoutBindMounts` then 😸

Reply:
If the error message is valid then the leading capital isn't the issue, but I'm at a loss to suggest what is.

Reply:
Weird:
```go
func TestSubTestNaming(t *testing.T) {
	t.Run("WithNameUpperCase", func(t *testing.T) {})
	t.Run("withNameLowerCase", func(t *testing.T) {})
}
```
=>
```console
=== RUN   TestSubTestNaming
=== RUN   TestSubTestNaming/WithNameUpperCase
=== RUN   TestSubTestNaming/withNamLowerCase
--- PASS: TestSubTestNaming (0.00s)
    --- PASS: TestSubTestNaming/WithNameUpperCase (0.00s)
    --- PASS: TestSubTestNaming/withNamLowerCase (0.00s)
PASS
coverage: 0.0% of statements
ok  	github.com/docker/app/internal/packager	0.022s	coverage: 0.0% of statements
Success: Tests passed.
```

Reply:
Honestly not sure what's going on here as it works perfectly find with other tests in that file

Reply:
The problem is that we use t.Name for naming containers within the test. Container names can't contain `/`.

Reply:
Yeah, this was a bit of a headscratcher, but it did indeed turn out to be a `docker-app` error message not a `go test` one. Phew!

Reply:
@zappy-shu I guess the error is fixed now? But my initial comment was about merging the 2 tests under the same top level test.